src/utils/config.ts

```diff
@@ -50,13 +50,14 @@ const testConfig = ({key, value}: ConfigUpdate): Nullable<string> => {
         case CONFIG_KEY.INITIAL_TAB_NAME:
             // This config option is not intended for direct user input.
             break;
-        case CONFIG_KEY.THEME:
-            throw new Error(`"${key}" cannot be managed using these utilities.`);
         case CONFIG_KEY.PAGE_SIZE:
             if (0 >= value || MAX_PAGE_SIZE < value) {
                 result = `Page size must be greater than 0 and less than ${MAX_PAGE_SIZE + 1}.`;
             }
             break;
+        case CONFIG_KEY.THEME:
```

Comment:
This was considered an impossible code path (i.e., calling `testConfig` or `setConfig` with `CONFIG_KEY.THEME` at any place would constitute an implementation error) because only JoyUI should have direct access on the theme config. We only created the theme-related types and localStorage keys to avoid type and key conflicts. With that said, i do see that we could be calling `setConfig` to set the theme once we implement the profile system, where we could be restoring theme name settings from a profile.
If you agree, we can implement `CONFIG_KEY.THEME` in the config utilities in preparation of the config profile system.